### DISCUSSION TITLE
Tab navigation slowish

### DISCUSSION THREAD

**Discussion by neonvoidx** [wezterm/wezterm#5143] [Q&A] [answered]
So I have wezterm setup like tmux now with multiplexing. In tmux I hit Ctrl +t as my leader then n or p to navigate forward or backwards . I setup wezterm to be the same, I have Ctrl t as my leader and then I have n and p setup to navigate forward or backwards tabs. 
For some reason it feels kind of slow to navigate between compared to tmux which is weird. It seems almost like I have to wait after I press control t for a second before I can press n or p or else it won't register my bind. Also it doesn't seem like I can press Ctrl t and spam n or p I have to fully let go. Is there a way to set this up better

**Comment by bew** [comment, marked as answer]
For more flexible 'modes' you can use Key Tables:
https://wezfurlong.org/wezterm/config/key-tables.html

**Reply by neonvoidx** [reply]
ok thanks, ill try this out
